pytest.ini

```diff
@@ -7,6 +7,5 @@
 # details.
 
 [pytest]
-addopts = --isort --pydocstyle --pycodestyle --doctest-glob="*.rst" --doctest-modules --cov=dictdiffer --cov-report=term-missing
```

Comment:
as run-tests.sh is used in CI, removing these here will prevent style and doctest checking in CI.

IMO the first three are a bit problematic as they depend on pytest plugins.

But they can be added to the requirements-dev.pip you are adding in this PR